src/queries.js

```diff
@@ -129,11 +165,40 @@ function queryByAltText(container, alt) {
 function getByAltText(container, alt) {
   const el = queryByAltText(container, alt)
   if (!el) {
-    throw new Error(`Unable to find an element with the alt text: ${alt}`)
+    const htmlElement = checkDebugLimit(
+      prettyFormat(container, {
+        plugins: [DOMElement],
+        printFunctionName: false,
+        highlight: true,
+      }),
+    )
+    throw new Error(
+      `Unable to find an element with the alt text: ${alt} \nHere is the state of your container: \n${htmlElement}`,
+    )
   }
   return el
 }
 
+function checkDebugLimit(debugContent) {
+  const limit = process.env.DEBUG_PRINT_LIMIT
+  if (limit) {
+    const contentLength = debugContent.length
+    debugContent = debugContent.slice(0, limit)
+    if (limit <= contentLength)
+      // there are more chars we have stripped, just show to the user
+      debugContent += '\n . . .'
+  } else {
+    // default limit is 7000
+    const contentLength = debugContent.length
+    debugContent = debugContent.slice(0, 7000)
+    if (contentLength >= 7000)
+      // there are more chars we have stripped, just show to the user
+      debugContent += '\n . . .'
```

Comment:
The above "if" and "else" are essentially copy-paste except `limit` is replaced with a default value.

I suggest pulling the "magic number" into a "defaultOptions" object and use it like so:
```
    const limit = defaultOptions.limit;
    // OR:  const limit = defaultOptions.limit || 7000;
    const contentLength = debugContent.length
    debugContent = debugContent.slice(0, limit)
    if (limit <= contentLength)
      // there are more chars we have stripped, just show to the user
      debugContent += '\n . . .'
```

Reply:
Yes, I agree. Can be improved. This is just a rough draft, I have to clean up few things. Thanks for your comments.